udata/frontend/csv.py

```diff
@@ -128,7 +152,11 @@ def nested_row(self, obj, nested):
         '''Convert an object into a flat csv row'''
         row = self.to_row(obj)
         for name, getter in self.get_nested_fields():
-            row.append(safestr(getter(nested)))
+            try:
+                row.append(safestr(getter(nested)))
+            except Exception, e:  # Catch all errors intentionally.
+                log.error('Error exporting CSV for {name}: {error}'.format(
```

Comment:
Same as above, will produce shorter row on error.
We need to append an empty value